src/main/java/io/xpring/ilp/model/PaymentResponse.java

```diff
@@ -7,10 +7,10 @@
  * A response object containing details about a requested payment
  */
 @Value.Immutable
-public interface SendPaymentResponse {
+public interface PaymentResponse {
```

Comment:
I'm fine with whatever name you want to use, but I think `PaymentStatus` or `PaymentResult` is more descriptive here.

Reply:
I think `PaymentResult` is probably the way to go.  `PaymentStatus` makes it sound like that status can change in the future.